### ISSUE TITLE
Bizarre compound/peaks mismatch in CASMI2016 data prepared with RMB

### ISSUE THREAD

**Issue by schymane** [MassBank/RMassBank#193]
Issue of mismatching peaks and compound info for 4 records/3 compounds in CASMI2016 data - TBD if this affects more records. Prepared using RMB, looks like (on first glance) a strange mismatch between msWorkflow and mbWorkflow. More details: 
https://github.com/MassBank/MassBank-data/issues/9

**Comment by meowcat** [comment]
Hey Emma,

this is not very good - however, in order to debug this in a meaningful way, I'd need the entire processing data for the CASMI records, i.e. mz(X)ML files, script, settings, input tables etc . Do we have that?

**Comment by schymane** [comment]
Yes I have this info but need to do the trace back my side to find the correct raw data etc before I can say more (it's complicated as it was compiled from several mixes). I'll keep you posted as soon as I know
more (been on the road...)

**Comment by meowcat** [comment]
@schymane: could you trace this back?
